[2024-01-21T11:31:08.006-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:31:05.770169+00:00 [queued]>
[2024-01-21T11:31:08.017-0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:31:05.770169+00:00 [queued]>
[2024-01-21T11:31:08.017-0800] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:31:08.017-0800] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-01-21T11:31:08.017-0800] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-21T11:31:08.063-0800] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): create_table> on 2024-01-21 19:31:05.770169+00:00
[2024-01-21T11:31:08.066-0800] {standard_task_runner.py:55} INFO - Started process 30155 to run task
[2024-01-21T11:31:08.086-0800] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'executing_sql_pipeline', 'create_table', 'manual__2024-01-21T19:31:05.770169+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/executing_sql_pipeline.py', '--cfg-path', '/tmp/tmpartddx9w']
[2024-01-21T11:31:08.088-0800] {standard_task_runner.py:83} INFO - Job 79: Subtask create_table
[2024-01-21T11:31:08.320-0800] {task_command.py:389} INFO - Running <TaskInstance: executing_sql_pipeline.create_table manual__2024-01-21T19:31:05.770169+00:00 [running]> on host Mittu.
[2024-01-21T11:31:08.473-0800] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=madhu
AIRFLOW_CTX_DAG_ID=executing_sql_pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-21T19:31:05.770169+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-21T19:31:05.770169+00:00
[2024-01-21T11:31:08.475-0800] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                    id INTEGER PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        
[2024-01-21T11:31:08.484-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:31:08.645-0800] {base.py:73} INFO - Using connection ID 'my_sqlite_conn' for task execution.
[2024-01-21T11:31:08.645-0800] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                    id INTEGER PRIMARY KEY,
                    name VARCHAR(50) NOT NULL,
                    age INTEGER NOT NULL,
                    is_active BOOLEAN DEFAULT true,
                    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
            );
        , parameters: None
[2024-01-21T11:31:08.655-0800] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=executing_sql_pipeline, task_id=create_table, execution_date=20240121T193105, start_date=20240121T193108, end_date=20240121T193108
[2024-01-21T11:31:08.714-0800] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-21T11:31:08.766-0800] {taskinstance.py:2596} INFO - 2 downstream tasks scheduled from follow-on schedule check
